### DISCUSSION TITLE
Exception: The read operation timed out for all my attempts

### DISCUSSION THREAD

**Discussion by ArielAssaf** [deepgram/community#631] [General help]
Here is the code I am running. I tried locally and in colab, burt i get the same error. I am trying to transcribe long .wav files that contain hebrew. Any help would be much appreciated. I am very new to coding and may be missing something basic. 

code:

from deepgram import (
    DeepgramClient,
    PrerecordedOptions,
    FileSource,
)

# Path to the audio file
AUDIO_FILE = "/content/R-00255_2402161950_VOS.WAV"

API_KEY = 'xxxxxxx'

def main():
    try:
        # STEP 1 Create a Deepgram client using the API key
        deepgram = DeepgramClient(API_KEY)

        with open(AUDIO_FILE, "rb") as file:
            buffer_data = file.read()

        payload: FileSource = {
            "buffer": buffer_data,
        }

        #STEP 2: Configure Deepgram options for audio analysis
        options = PrerecordedOptions(
            model="whisper-medium",
            smart_format=True,
            language="he",
            diarize=True,
        )

        # STEP 3: Call the transcribe_file method with the text payload and options
        response = deepgram.listen.prerecorded.v("1").transcribe_file(payload, options)

        # STEP 4: Print the response
        print(response.to_json(indent=4))

    except Exception as e:
        print(f"Exception: {e}")

if __name__ == "__main__":
    main()

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
